projects/GEDKeeper2/GKUI/BaseWin.cs

```diff
@@ -1376,55 +1455,119 @@ public bool RecordDelete(GEDCOMRecord record, bool confirm)
             {
                 //string xref = record.XRef;
                 string msg = "";
-
                 switch (record.RecordType)
                 {
                     case GEDCOMRecordType.rtIndividual:
-                        msg = string.Format(LangMan.LS(LSID.LSID_PersonDeleteQuery), ((GEDCOMIndividualRecord)record).GetNameString(true, false));
+                    {
+                        msg = string.Format(LangMan.LS(
+                            LSID.LSID_PersonDeleteQuery),
+                            ((GEDCOMIndividualRecord)record).GetNameString(
+                                true, false));
                         break;
+                    }
 
                     case GEDCOMRecordType.rtFamily:
-                        msg = string.Format(LangMan.LS(LSID.LSID_FamilyDeleteQuery), GKUtils.GetFamilyString((GEDCOMFamilyRecord)record));
+                    {
+                        msg = string.Format(LangMan.LS(
+                            LSID.LSID_FamilyDeleteQuery),
+                            GKUtils.GetFamilyString(
+                                (GEDCOMFamilyRecord)record));
                         break;
+                    }
 
                     case GEDCOMRecordType.rtNote:
-                        msg = LangMan.LS(LSID.LSID_NoteDeleteQuery);
+                    {
+                        StringList l = ((GEDCOMNoteRecord) (record)).Note;
+                        string value;
+                        if (0 != l.Count)
+                        {
+                            if (NoteNameMaxLength < l[0].Length)
+                            {
+                                value = l[0].Substring(0, NoteNameMaxLength) +
+                                        "...";
+                            }
+                            else
+                            {
+                                value = l[0];
+                            }
+                        }
+                        else
+                        {
+                            value = string.Format("#{0}",
+                                                  record.GetId().ToString());
+                        }

```

Comment:
It is better to make a separate function (in GKUtils) - it can be embedded in one or two places.